test/trento/sap_systems/projections/sap_system_projector_test.exs

```diff
@@ -335,7 +335,7 @@ defmodule Trento.SapSystems.Projections.SapSystemProjectorTest do
       projection =
       SapSystemReadModel
       |> Repo.get(sap_system_id)
-      |> Repo.preload([:tags])
+      |> Repo.preload([:tags, :database])
```

Comment:
we should change the assert on this test, right?
the `database_sid` should be there, in the broadcast